### PR TITLE
Adding BaseUnits for a handful of quantities

### PR DESCRIPTION
**Pull request by tmilnthorp**

### CODE REVIEW

File: Common/UnitDefinitions/Force.json [hunk 1]
```diff
@@ -11,6 +11,11 @@
     {
       "SingularName": "Dyn",
       "PluralName": "Dyne",
+      "BaseUnits": {
+        "L": "Centimeter",
```
Comment (angularsen) [inline Common/UnitDefinitions/Force.json]: Where do you look up these facts? I have no prior reference. Do you google each you are unsure about or do you have a goto-source? I found this one on wiki.

File: Common/UnitDefinitions/Force.json [hunk 2]
```diff
@@ -93,6 +103,11 @@
     {
       "SingularName": "Poundal",
```
Comment (angularsen) [inline Common/UnitDefinitions/Force.json]: I see there is a notion of foot-pound-second system and centimeter-gram-second system (CGS / Gauss).
I'll create a PR to discuss adding those.

**Comment by angularsen** [comment]
Looks good to me

File: Common/UnitDefinitions/Force.json (hunk 1 above)
Comment (tmilnthorp) [reply Common/UnitDefinitions/Force.json]: Here's [one ](https://en.wikipedia.org/wiki/Dyne) for dyne:
> The dyne is defined as "the force required to accelerate a mass of one gram at a rate of one centimetre per second per second"

Or try [this](https://www.wolframalpha.com/input/?i=dyne):

> Unit system:
centimeter-gram-second (cgs)

File: Common/UnitDefinitions/Force.json (hunk 2 above)
Comment (tmilnthorp) [reply Common/UnitDefinitions/Force.json]: Indeed there is. CGS, FPS, MKS

File: Common/UnitDefinitions/Force.json (hunk 1 above)
Comment (angularsen) [reply Common/UnitDefinitions/Force.json]: Thanks

**Pull request merged**